### ISSUE TITLE
[Feature Request] Bring Edit view of class comments on double click

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#12822]
Currently to edit a class comment you need to go toggle the "edit view" checkbox that is at the bottom right of the screen. 

It would be great to be able to just double click on the class comment to toggle the edit view

**Comment by Ducasse** [comment]
I agree but I do not know how to do it else I would have done it already

**Comment by jecisc** [comment]
I’ll try to take a look this week

**Comment by hernanmd** [comment]
FYI you could change between view and edit modes with CMD+R.

**Comment by jecisc** [comment]
@AlexisCnockaert This would improve the UX of Pharo I think :)

**Comment by jecisc** [comment]
@AlexisCnockaert Now that you know Pharo a little better, maybe this issue could be nice to do

**Comment by AlexisCnockaert** [comment]
I will take care of it Cyril
